docs/carts/cart_shop.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Shop pages
+parent: Carts
+nav_order: 1
+---
+
+# Easol shop pages
+In addition to a [product page]({% link docs/theme_architecture/templates.md %}), each experience has an automatically generated 'shop' page, which can be found at `site.com/shop/product-slug`. 
+
+This page includes key details of the experience including the name, country, dates, tagline and the hero image. On this page customers are able to add items to their cart, remove items from their cart, update item quantities and see a breakdown of the contents of their cart before proceeding to checkout.
+
+## Shop page customisation
+The Creator's [brand colours]({% link docs/site_styles/index.md %}) set under My Site > Styles are applied to some elements of the shop page by default. 
+
+You are not able to amend the html or add additional javascript to shop pages. However, further customisations can be applied to the shop page by adding additional styles into the [index.css]({% link docs/theme_architecture/assets.md %}) file of a theme, using selector `.cart-shop`.
```

Comment:
Nice! Is it worth mentioning how to avoid styles leaking into the cart shop or have we solved for that another way? The example I'm remembering is Meadows in the mountains having round yellow buttons with little pigs(?) on in the cart shop, matching their Site.

Reply:
Good thinking - I've added additional details